### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by keirthana**
This PR adds the security policy to our Anbox Cloud docs repository

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Anbox Cloud security policy
```
Comment (morphis) [inline SECURITY.md:1]: Should we put the same into our documentation given that most components aren't open source and wont have a SECURITY.md?

**Review by morphis — APPROVED** [review]
LGTM

File: SECURITY.md (hunk 1 above)
Comment (keirthana) [reply SECURITY.md:1]: Couldn't hurt to put it in docs. I will create a PR for that.

**Review by adglkh — APPROVED** [review]
LGTM

**Pull request merged**